assets/tiled-project/tilesets/jm-animations.tsx

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you please rename the file to give it understandable name - not sure what 'jm-' stands for.

Reply:
jm are my initials, but I agree that it is better to choose a name that is more understandable